rpcs3/Emu/Cell/lv2/sys_memory.cpp

```diff
@@ -43,6 +45,12 @@ error_code sys_memory_allocate(u32 size, u64 flags, vm::ptr<u32> alloc_addr)
 		return CELL_ENOMEM;
 	}
 
+	if (!alloc_addr)
+	{
+		dct->used -= size;
+		return {CELL_EFAULT, alloc_addr};
```

Comment:
For the sake of nitpicking, it's better to not use second argument because it's clearly zero.